File: app/src/main/java/com/onemoresecret/HiddenTextFragment.java
package com.onemoresecret;

import android.os.Bundle;
import android.util.Log;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Toast;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.onemoresecret.crypto.OneTimePassword;
import com.onemoresecret.databinding.FragmentHiddenTextBinding;
import com.onemoresecret.databinding.FragmentTotpBinding;

import java.util.Timer;
import java.util.TimerTask;
import java.util.function.Consumer;
import java.util.function.Supplier;

public class HiddenTextFragment extends Fragment {
    private FragmentHiddenTextBinding binding;

    @Override
    public View onCreateView(
            @NonNull LayoutInflater inflater, ViewGroup container,
            Bundle savedInstanceState) {
        binding = FragmentHiddenTextBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }

    public void setText(String text) {
        binding.textViewMessage.setText(text);
    }
}

File: app/src/main/java/com/onemoresecret/HiddenTextFragment.kt
package com.onemoresecret;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import com.onemoresecret.databinding.FragmentHiddenTextBinding;

public class HiddenTextFragment extends Fragment {
    private FragmentHiddenTextBinding binding;

    @Override
    public View onCreateView(
            @NonNull LayoutInflater inflater, ViewGroup container,
            Bundle savedInstanceState) {
        binding = FragmentHiddenTextBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }

    public void setText(String text) {
        binding.textViewMessage.setText(text);
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        binding.textViewMessage.setText(getString(R.string.hidden_text));
        binding = null;
    }
}